speech/google/cloud/speech/sample.py

```diff
@@ -47,8 +47,8 @@ class Sample(object):
     default_encoding = Encoding.FLAC
     default_sample_rate = 16000
 
-    def __init__(self, content=None, source_uri=None,
-                 encoding=None, sample_rate=None):
+    def __init__(self, content=None, source_uri=None, encoding=None,
+                 sample_rate=None):
```

Comment:
Yeah I had another input type `stream` but I took it out and reformatted. Apparently this was wrongly formatted in `master`.